pkg/controller/controller.go

```diff
@@ -90,6 +104,14 @@ func AddToManager(ctx context.Context,
 				return nil
 			},
 		},
+		{
+			flightctl.NamespaceControllerName,
```

Comment:
I still concern the idea of starting these controllers by default. It brings risk to our normal import flow when flightctl is not enabled.

Reply:
If I understanding it right, another way is to use backplane-operator to tell the import-controller if flightctl is enabled or not at the pod start phase.  

I added a question into the doc, we may need to check the progress of flightctl + MCE.

Reply:
Need to confirm with the installer side, see if they have plan and buffer to implement a flag we can use.

Reply:
Updated to use a flag to control everything related to flightctl.